.meta.toml

```
@@ -0,0 +1,21 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+[meta]
+template = "pure-python"
+commit-id = "375a92164048640ea5892fdd34444a55d9ebd299"
+
+[python]
+with-pypy = true
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 0
```

Here should be some kind of baseline.
```suggestion
fail-under = 31
```